config/initializers/sheerid_ip_whitelist.rb

```diff
@@ -0,0 +1,11 @@
+SHEERID_IP_WHITELIST = [
+  # production environment
+  '54.236.234.189',
```

Comment:
you really want a whitelist committed to git?

Reply:
I don't think it's the worst thing in the world. I believe this is public information anyway.